docs/resources/specification.md

```diff
@@ -181,20 +181,24 @@ A set of [Tag](#Tag) instances.
 
 A `Tag` can add domain specific metadata to an event allowing for custom partitioning
 
-!!! note
+!!! info
     Usually a tag represents a concept of the domain, e.g. the type and id of an entity like `product:p123`
 
 - It _MUST_ satisfy the regular expression `/^[[:alnum:]\-\_\:]{1,150}`
+- It _CAN_ represent a key/value pair such as `product:123` but that is irrelevant to the Event Store
 
 ### AppendCondition
 
 - It _MUST_ contain a [StreamQuery](#StreamQuery)
-- It _MUST_ contain a [ExpectedHighestSequenceNumber](#ExpectedHighestSequenceNumber)
-
-### ExpectedHighestSequenceNumber
```

Comment:
What about `SafePoint`?

Reply:
I think I like that idea!

Reply:
We agreed to calling it `expectedCeiling` for now (although I must confess that I am not super satisfied with it yet)